helm/amko/crds/gslbhostrule_def.yaml

```diff
@@ -32,14 +32,28 @@ spec:
                 minimum: 0
               sitePersistence:
                 description: "Maintain stickiness to the same site where the connection was initiated."
-                type: object
-                properties:
-                  enabled:
-                    description: "Enable/Disable GSLB site persistence"
-                    type: boolean
-                  profileRef:
-                    description: "Custom site persistence profile reference, will be applied on the GSLB Service"
-                    type: string
+                type: array
```

Comment:
Is there a reason why we are making this an array? The Gslb Service object doesn't take an array for site persistence:
```
    {
      "_last_modified": "string",
      "application_persistence_profile_ref": "string",
      "controller_health_status_enabled": true,
```

Reply:
As discussed, it should be an object not array. Adding the necessary changes